lmnet/lmnet/datasets/dataset_iterator.py

```diff
@@ -208,8 +211,17 @@ def __init__(self, dataset, enable_prefetch=False, seed=0):
         self.seed = seed
 
         if self.enable_prefetch:
-            self.prefetch_result_queue = queue.Queue(maxsize=200)
-            self.prefetcher = _MultiProcessDatasetPrefetchThread(self.dataset, self.prefetch_result_queue, seed)
+            queue_size = self.dataset.batch_size
```

Comment:
queue size is not related to barch_size. Please change.

Reply:
@ruimashita 
I've updated queue size. Updated code for read from dataset config. if set it will use it or set to 50